### DISCUSSION TITLE
Upcoming features to allow non-ASCII labels (id in HTML tags)

### DISCUSSION THREAD

**Discussion by briandfoy** [briandfoy/text-multimarkdown#9] [Announcements]
I'm working toward a new version to satisfy the request in #6 for labels to either transliterate non-ASCII to ASCII or just use full on Unicode labels. These add the new options `transliterated_ids` and `unicode_ids`.

It's unfortunate that the parent class, [Text::Markdown](https://www.metacpan.org/pod/Text::Markdown) chose to name its function to produce HTML `markdown`. That's not disappearing. Also, it's extra frustrating that `markdown` is also a method. And that it has an optional argument.

`markdown` isn't disappearing because downstreams are using it, but I'm working on some alternative names that better represent what it does.

* In the functional form, `multimarkdown_to_html`.
* In the object form, `to_html`

Nothing is released yet and there is plenty of time to make changes and to test against your own distributions. I'd like to try out GitHub discussions to give downstreams fair warning and a chance to weigh in before anything happens.

**Comment by cxw42** [comment]
+1 to `$m->to_html`
